src/formGenerator.vue

```diff
@@ -301,10 +310,16 @@ div.vue-form-generator(v-if='schema != null')
 			// Validating the model properties
 			validate() {
 				this.clearValidationErrors();
+				this.touched.splice(0);
 
 				this.$children.forEach((child) => {
 					if (isFunction(child.validate))
 					{
+						this.touched.push({
```

Comment:
It's a special data type/class which represents a component or a tree of elements in vue
https://github.com/vuejs/vue/blob/dev/src/core/vdom/vnode.js
It basically contains the element, tag, component data and many other things. Since slots return a list of `$vnodes` then you can do the extra check and see if the component you are trying to add exists in slots if slots are used. Since vfg uses `component :is`, then the filtering probably can be done based on `$children` maybe